pkg/mapper/mapper_benchmark_test.go

```diff
@@ -582,9 +585,18 @@ mappings:` + duplicateRules(100, ruleTemplateSingleMatchGlob)
 	}
 
 	for _, cacheType := range []string{"lru", "random"} {
+		mapper := MetricMapper{}
+		var cache MetricMapperCache
+		switch cacheType {
+		case "lru":
+			cache, _ = lru.NewMetricMapperLRUCache(mapper.Registerer, 1000)
+		case "random":
+			cache, _ = randomreplacement.NewMetricMapperRRCache(mapper.Registerer, 1000)
+		}
```

Comment:
a bunch of tests are repeating this. would it be easier to have a helper that returns a (fresh) `map[string]MetricMapperCache` so we can `range cacheType, cache := cacheTypes() { … }`?

Reply:
could we generalize things if that also includes the `noop` cache?

Reply:
yeah, I'll add a helper function for this.